.github/workflows/check-broken-pr-links.yml

```
@@ -0,0 +1,120 @@
+      # 1️⃣ Checkout repository
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      # 2️⃣ Get changed Markdown files in the PR
+      - name: Get changed Markdown files
+        id: changed-files
+        run: |
+          CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.sha }} | grep '\.md$' || true)
+          CHANGED_FILES="${CHANGED_FILES//$'\n'/ }"
+          echo "CHANGED_FILES=$CHANGED_FILES" >> $GITHUB_ENV
+          echo "Changed Markdown files: $CHANGED_FILES"
+
+      # 3️⃣ Skip if no Markdown files changed
+      - name: Skip if no Markdown files changed
+        if: env.CHANGED_FILES == ''
+        run: |
+          echo "No Markdown files changed. Skipping link check."
+          exit 0
+
+      # 4️⃣ Run Lychee on changed files
+      - name: Run Lychee
+        id: run-lychee
+        uses: lycheeverse/lychee-action@v2
+        with:
+          args: |
+            --no-progress
+            --include-fragments
+            --format detailed
+            ${{ env.CHANGED_FILES }}
+          output: lychee/out_raw.md
+          fail: false  # ✅ don't fail yet, let us capture output
+
+      # 5️⃣ Format Lychee output (user-friendly, relative paths)
+      - name: Format Lychee report
+        id: format-report
+        if: always()
+        run: |
+          mkdir -p lychee
+          : > lychee/comment.md   # start with empty file
+      
+          awk '
+            /^Errors in / {
+              file=$3
+              gsub("^/home/runner/work/UmbracoDocs/UmbracoDocs/", "", file)
+              print "\nFile: " file >> "lychee/comment.md"
+              next
+            }
+      
+            /\[ERROR\]/ {
+              msg = $0
+              sub(/^- \[ \] /, "", msg)
+              sub(/^\[ERROR\] /, "", msg)
+              gsub("^file:///home/runner/work/UmbracoDocs/UmbracoDocs/", "", msg)
+              print "⚓ Anchor not found → " msg >> "lychee/comment.md"
+              next
+            }
+      
+            /\[404\]/ {
+              msg = $0
+              sub(/^- \[ \] /, "", msg)
+              sub(/^\[404\] /, "", msg)
+              print "❌ 404 Not Found → " msg >> "lychee/comment.md"
+              next
+            }
+      
+            /\[301\]|\[302\]/ {
+              msg = $0
+              sub(/^- \[ \] /, "", msg)
+              sub(/^\[(301|302)\] /, "", msg)
+              print "🔀 Redirect → " msg >> "lychee/comment.md"
+              next
+            }
+      
+            /Timeout/ && !/Timeouts/ {
+              msg = $0
+              sub(/^- \[ \] /, "", msg)
+              print "⏳ Timeout → " msg >> "lychee/comment.md"
+              next
+            }
+          ' lychee/out_raw.md
+      
+          # Add header only if we found content
+          if [ -s lychee/comment.md ]; then
+            sed -i '1i 🚨 **Link Checker found broken links in this PR**\n' lychee/comment.md
```

Made the header a little friendlier. Also broke it up into two lines for readability.

```suggestion
            sed -i '1i **The Link Checker found broken links in your PR**.\n Please review the following list:\n' lychee/comment.md
```